packages/hardhat/src/utils/hardhat.js

```diff
@@ -1,5 +1,3 @@
-const hre = require('hardhat');
```

Comment:
You can't do this in anything used by a task (tasks manage their own hre as a parameter).